lang/python/python3/patches/016-adjust-config-paths.patch

```diff
@@ -20,16 +20,16 @@
          abi=sys.abiflags,
 --- a/Makefile.pre.in
 +++ b/Makefile.pre.in
-@@ -1460,7 +1460,7 @@ libinstall:	build_all $(srcdir)/Modules/
+@@ -1521,7 +1521,7 @@ libinstall:	build_all $(srcdir)/Modules/xxmodule.c
  			esac; \
  		done; \
  	done
 -	$(INSTALL_DATA) `cat pybuilddir.txt`/_sysconfigdata_$(ABIFLAGS)_$(MACHDEP)_$(MULTIARCH).py \
 +	$(INSTALL_DATA) `cat pybuilddir.txt`/_sysconfigdata*.py \
  		$(DESTDIR)$(LIBDEST); \
  	$(INSTALL_DATA) $(srcdir)/LICENSE $(DESTDIR)$(LIBDEST)/LICENSE.txt
- ifeq (@COMPILE_ALL_TESTS@,yes)
-@@ -1618,7 +1618,7 @@ sharedinstall: sharedmods
+ 	if test -d $(DESTDIR)$(LIBDEST)/distutils/tests; then \
```

Comment:
Was this change intentional?

Reply:
umm, let me re-apply it;
the patch context may have changed;

Reply:
ok; so it's semi-intentional; i'll update this;
this shows up because i applied this patch on a Python-3.9 folder without applying the previous patches; so the context did not update;

Reply:
thanks for catching this :)